### ISSUE TITLE
Excluded paths are checked and counted

### ISSUE THREAD

**Issue by konstruktoid** [ansible/ansible-lint#4517]
##### Summary

From https://github.com/konstruktoid/ansible-role-hardening/actions/runs/13222601423/job/36909541472#step:6:424
`Excluded: molecule/default/molecule.yml` but
```
yaml[line-length]: Line too long (190 > 160 characters)
molecule/default/molecule.yml:15
```

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

https://github.com/konstruktoid/ansible-role-hardening/blob/master/action-lint/Dockerfile
https://github.com/konstruktoid/ansible-role-hardening/actions/runs/13222601423/job/36909541472

`python3 -m pip install --break-system-packages --no-cache-dir --upgrade jmespath passlib git+https://github.com/ansible/ansible-lint.git`

##### STEPS TO REPRODUCE
```
$ bash action-lint/entrypoint.sh
# Running ansible-lint
ansible-lint 25.1.1 using ansible-core:2.18.2 ansible-compat:25.1.1 ruamel-yaml:0.18.10 ruamel-yaml-clib:0.2.12
A new release of ansible-lint is available: 25.1.1 → 25.1.2[/] Upgrade by running: pip install --upgrade ansible-lint
[...]
INFO     Loading ignores from .gitignore
INFO     Excluded: .git
INFO     Excluded: .github
INFO     Excluded: .pre-commit-config.yaml
INFO     Excluded: .vagrant
INFO     Excluded: molecule/almalinux/molecule.yml
INFO     Excluded: molecule/custom/molecule.yml
INFO     Excluded: molecule/debian/molecule.yml
INFO     Excluded: molecule/default/molecule.yml
INFO     Excluded: molecule/docker/molecule.yml
INFO     Excluded: molecule/single/molecule.yml
INFO     Excluded: molecule/ubuntu/molecule.yml
INFO     Executing syntax check on playbook molecule/default/converge.yml (0.52s)
INFO     Executing syntax check on playbook molecule/default/verify.yml (0.61s)
INFO     Executing syntax check on playbook tests/test.yml (1.08s)

Passed: 0 failure(s), 0 warning(s) on 156 files. Profile 'production' was required, and it passed.
A new release of ansible-lint is available: 25.1.1 → 25.1.2[/] Upgrade by running: pip install --upgrade ansible-lint
```

The fails with `25.1.2`.

##### Desired Behavior

Excluded paths should be excluded.

##### Actual Behavior

```sh
$ bash action-lint/entrypoint.sh
# Running ansible-lint
ansible-lint 25.1.2 using ansible-core:2.18.2 ansible-compat:25.1.2 ruamel-yaml:0.18.10 ruamel-yaml-clib:0.2.12
[...]
INFO     Excluded: molecule/almalinux/molecule.yml
INFO     Excluded: molecule/custom/molecule.yml
INFO     Excluded: molecule/debian/molecule.yml
INFO     Excluded: molecule/default/molecule.yml
INFO     Excluded: molecule/docker/molecule.yml
INFO     Excluded: molecule/single/molecule.yml
INFO     Excluded: molecule/ubuntu/molecule.yml
INFO     Executing syntax check on playbook molecule/default/converge.yml (0.64s)
INFO     Executing syntax check on playbook molecule/default/verify.yml (0.75s)
INFO     Executing syntax check on playbook tests/test.yml (0.87s)
WARNING  Listing 7 violation(s) that are fatal
Read documentation for instructions on how to ignore specific rule violations.

# Rule Violation Summary

  7 yaml profile:basic tags:formatting,yaml

Failed: 7 failure(s), 0 warning(s) on 164 files. Profile 'production' was required, but 'min' profile passed.
yaml[line-length]: Line too long (190 > 160 characters)
molecule/almalinux/molecule.yml:15

yaml[line-length]: Line too long (190 > 160 characters)
molecule/custom/molecule.yml:15

yaml[line-length]: Line too long (190 > 160 characters)
molecule/debian/molecule.yml:15

yaml[line-length]: Line too long (190 > 160 characters)
molecule/default/molecule.yml:15

yaml[line-length]: Line too long (190 > 160 characters)
molecule/docker/molecule.yml:13

yaml[line-length]: Line too long (190 > 160 characters)
molecule/single/molecule.yml:15

yaml[line-length]: Line too long (190 > 160 characters)
molecule/ubuntu/molecule.yml:15

ansible-lint failed.
```
